sorc/grid_tools.fd/regional_esg_grid.fd/psym2.f90

```diff
@@ -35,18 +36,16 @@ module psym2
 
 contains
 
-!=============================================================================
+!> Get the orthogonal eigenvectors, vv, and diagonal matrix of eigenvalues, oo,
+!! of the symmetric 2*2 matrix, em.
+!! @param em symmetric 2*2 matrix
+!! @param vv orthogonal eigenvectors
+!! @param oo diagonal matrix of eigenvalues
```

Comment:
Put an author tag on each function/subroutine. If the function/subroutine doesn't have an author, use the one from the file level. If the file doesn't have one either, ask.